diffsync/store/redis.py

```diff
@@ -226,17 +208,21 @@ def remove(self, obj: "DiffSyncModel", remove_children: bool = False):
             for child_type, child_fieldname in obj.get_children_mapping().items():
                 for child_id in getattr(obj, child_fieldname):
                     try:
-                        child_obj = self.get(child_type, child_id)
-                        self.remove(child_obj, remove_children=remove_children)
+                        child_obj = self.get(model=child_type, identifier=child_id)
+                        self.remove(obj=child_obj, remove_children=remove_children)
                     except ObjectNotFound:
                         pass
                         # Since this is "cleanup" code, log an error and continue, instead of letting the exception raise
                         # self._log.error(f"Unable to remove child {child_id} of {modelname} {uid} - not found!")
 
-    def count(self, modelname=None):
-        """Returns the number of elements of an specific model name."""
+    def count(self, *, model: Union[Text, "DiffSyncModel", Type["DiffSyncModel"], None] = None) -> int:
+        """Returns the number of elements of a specific model, or all elements in the store if unspecified."""
         search_pattern = f"{self._store_label}:*"
-        if modelname:
+        if model:
```

Comment:
I think I remember some bug in the past where either a DiffSyncModel or a DiffSync was evaluating as falsey, resulting in unexpected behavior. Perhaps:

```suggestion
        if model is not None:
```